routes/index.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Set up routes for save/publish
+*/
+
+module.exports = function(server) {
+
+  // /projects/* API routes
+  // Handles user-level operations
+  require('./projects/get-projects.js')(server);
+  require('./projects/create-project.js')(server);
+  require('./projects/delete-project.js')(server);
+  require('./projects/publish-project.js')(server);
+
+  // /files/* API routes
+  // Handles project-level operations
+  require('./files/get-project-contents.js')(server);
+  require('./files/get-file-contents.js')(server);
```

Comment:
i.e.:
* `projects/:username` - returns a list of projects
* `project/:username/:projectName/*` - crud and publish **a** project
* `file/:username/:projectName/*` - crud on files in a project

Is that what you have in mind?